chat.delta.desktop.tauri.yml

```diff
@@ -0,0 +1,136 @@
+app-id: chat.delta.desktop.tauri
```

Comment:
this is not an acceptable app-id. "Tauri" is not the application name. (Nor "desktop", the `chat.delta.desktop` would be rejected today, for the record)

Reply:
would `chat.delta.DeltaChatTauri` be good?

Reply:
in general the name of the toolkit in the application name is bad taste. But that's your problem.